src/index.css

```diff
@@ -0,0 +1,5 @@
+.h1 {
```

Comment:
The selector '.h1' seems incorrect. It is not recommended to use HTML tags as class names. Consider using a more descriptive class name that reflects the purpose or context of the element.